promgen/templates/promgen/permission_row.html

```diff
@@ -20,6 +20,21 @@
       {{ perm|upper }}
     </span>
     {% endfor %}
+
+    <button class="btn btn-xs" data-toggle="collapse" data-target="#user{{user.id}}">
```

Comment:
I think it will look better if this button does not have a background. Can you try it?

```suggestion
    <button style="background:none" class="btn btn-xs" data-toggle="collapse" data-target="#user{{user.id}}">
```